docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -181,8 +181,8 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `SSH_LISTEN_PORT`: **%(SSH\_PORT)s**: Port for the built-in SSH server.
 - `OFFLINE_MODE`: **false**: Disables use of CDN for static files and Gravatar for profile pictures.
 - `DISABLE_ROUTER_LOG`: **false**: Mute printing of the router log.
-- `CERT_FILE`: **custom/https/cert.pem**: Cert file path used for HTTPS.
-- `KEY_FILE`: **custom/https/key.pem**: Key file path used for HTTPS.
+- `CERT_FILE`: **https/cert.pem**: Cert file path used for HTTPS. From 1.12 relative paths will be made absolute against `CUSTOM_PATH`.
```

Comment:
1.11? Since ther will be a backport to v1.11?

Reply:
Yeah if you're happy for it to be backported

Reply:
```suggestion
- `CERT_FILE`: **https/cert.pem**: Cert file path used for HTTPS. From 1.11 relative paths will be made absolute against `CUSTOM_PATH`.
```